Dokan/DokanTests/Scenes/ReviewProduct/RatingDetailsViewModelTests.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  RatingDetailsViewModelTests.swift
+//  DokanTests
+//
+//  Created by marwa on 31/08/2022.
+//
+@testable import Dokan
+import XCTest
+
+class RatingDetailsViewModelTests: XCTestCase {
+
+    // MARK: - Properties
+
+    var sut: RatingDetailsViewModel!
+
+    // MARK: - life cycle
+
+    override func setUpWithError() throws {
+        sut = RatingDetailsViewModel(rating: 4.3,
+                                     fiveStarNumber: 6,
+                                     fourStarNumber: 3,
+                                     threeStarNumber: 5,
+                                     twoStarNumber: 4,
+                                     oneStarNumber: 2,
+                                     totalReviewsNumber: 20)
+    }
+
+    override func tearDownWithError() throws {
+        sut = nil
+    }
+
+    func testRating_WhenCallConfigureRatingLabelFunction_ReturnString() {
+        // When
+        let rating = sut.configureRatingLabel()
+        // Then
```

Comment:
Let's keep a space before each step for better readability. This applies to add cases.
```suggestion
        
        // Then
```